hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -933,6 +933,10 @@ public boolean getGlobalSimpleIndexUpdatePartitionPath() {
     return getBoolean(HoodieIndexConfig.SIMPLE_INDEX_UPDATE_PARTITION_PATH);
   }
 
+  public int getNumBuckets() {
+    return Integer.parseInt(props.getProperty(HoodieIndexConfig.BUCKET_INDEX_BUCKET_NUM.key(), "-1"));
```

Comment:
Can we use the `BUCKET_INDEX_BUCKET_NUM.defaultValue()` here

Reply:
fix